crates/typst-ide/src/complete.rs

```diff
@@ -923,10 +925,17 @@ fn complete_code(ctx: &mut CompletionContext) -> bool {
     }
 
     // Anywhere: "{ | }".
```

Comment:
More examples to help understand the code:

```suggestion
    // Anywhere: "{ | }", "(|)", "(1,|)", "(a:|)".
```